src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,30 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistrbucion(val ciudad: Ciudad){
+    val vendedores = mutableListOf<Vendedor>()
+
+    fun agregarVendedor(vendedor: Vendedor) {
+        if (vendedores.contains(vendedor)){
+            throw Exception("Ya está registrado en el centro")
+        }
+        else{
+            vendedores.add(vendedor)
+        }
```

Comment:
Si se ejecuta la rama verdadera del `if`, el método explota y nunca llega al `else`. Podés entonces obviarlo, te quedaría así:

```suggestion
        if (vendedores.contains(vendedor)){
            throw Exception("Ya está registrado en el centro")
        }
        
        vendedores.add(vendedor)
```